### PR TITLE
PR CI run only compiles, ignores actual tests

### PR DESCRIPTION
**Pull request by pjfanning**
Nightly job to run all tests

### CODE REVIEW

File: .github/workflows/nightly-builds.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      - name: Set up JDK 8
+        uses: coursier/setup-action@v1.2.1
+        with:
+          jvm: adopt:8
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.4.0
+
+      - name: "Run all tests. Run locally with: sbt +~2.13 \"test\""
+        run: sbt +~2.13 "test"
```
Comment (mdedetrich) [inline .github/workflows/nightly-builds.yml:34]: What does this `~` syntax do exactly?

Comment (pjfanning) [reply .github/workflows/nightly-builds.yml:34]: not sure - I just copied it from the existing workflow yml - I can remove it

Comment (mdedetrich) [reply .github/workflows/nightly-builds.yml:34]: I would say leave it in there, was just curious. Ill approve the PR

**Review by mdedetrich — APPROVED** [review]
lgtm

**Pull request merged**